convert2rhel/checks.py

```diff
@@ -443,8 +443,15 @@ def get_rhel_supported_kmods():
     #   platform ID.
     # get output of a command to get all packages which are the source
     # of kmods
-    kmod_pkgs_str, _ = run_subprocess(cmd, print_output=False)
+    kmod_pkgs_str, repoquery_exit_code = run_subprocess(cmd, print_output=False)
 
+    if repoquery_exit_code != 0:
+        logger.critical(
+            "We were unable to query the repositories (%s) to"
+            " determine packages containing kernel modules."
+            " Output from the failed repoquery command:\n\n%s"
+            % (", ".join(system_info.get_enabled_rhel_repos()), kmod_pkgs_str)
+        )
```

Comment:
We should unit test this due to it being a critical exit

Reply:
<nod>.  Done.